### PR TITLE
Pupil community blogpost

### PR DESCRIPTION
**Pull request by nathakits**
add new pupil community blogpost

### CODE REVIEW

File: contents/articles/2018-01_pupil_community/index.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+- Computer Controlling System using Eye-Tracking and Voice Recognition
+
+### Lab Matlab Control 
+https://github.com/TheGoldLab/Lab-Matlab-Control/tree/62d56585ef2fda3c17045dfcdc69e159eb317a38
+
+- Custom utilities for experimental control for the Gold Lab
+
+
+Our Open source software is free for you to use and explore. Make a suggestion, fix a bug, improve documentation, or contribute code to a project. Get involved and contribute and be part of Pupil. Submit your project to the github repository by creating a PR to the list.
```
Comment (mkassner) [inline contents/articles/2018-01_pupil_community/index.md]: what does this mean? What repo? Link?

File: contents/articles/2018-01_pupil_community/index.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+subtitle: "Pupil is a community driven development project. Check out what the community has been doing with Pupil..."
+featured_img: ""
+featured_img_thumb: ""
+---
+
+Happy 2018!
+Our first blog post of the year will be about the Pupil community, looking at how much Pupil has grown!
+Pupil is a community driven development project to make eye tracking hardware that is accessible, hackable, affordable and software that is open source.
+
+Check out what the community has been doing! Here are a list of custom tools and projects built with Pupil.
```
Comment (mkassner) [inline contents/articles/2018-01_pupil_community/index.md]: add something to state that this is by no means complete and there may be many more out there!

**Comment by mkassner** [comment]
Please see comments before merging.

File: contents/articles/2018-01_pupil_community/index.md (hunk 1 above)
Comment (nathakits) [reply contents/articles/2018-01_pupil_community/index.md]: Me and @willpatera were discussing making a blog post about the tools and projects the community has done. We thought of making a new repo that is a list of all the projects we've found and that other people who has developed something with pupil can add their project to the list via PRs.

File: contents/articles/2018-01_pupil_community/index.md (hunk 2 above)
Comment (nathakits) [reply contents/articles/2018-01_pupil_community/index.md]: Yes, i will make the change.

File: contents/articles/2018-01_pupil_community/index.md (hunk 1 above)
Comment (mkassner) [reply contents/articles/2018-01_pupil_community/index.md]: sounds good. I understand now that this is in WIP mode. A repo with one page and PR contributions sounds good!

Comment (willpatera) [reply contents/articles/2018-01_pupil_community/index.md]: @mkassner I was considering making a new repository that has only a Readme file in the repository where community members can submit a PR to include their project, plugin, or script. We can use this as a resource to showcase community contributed work and as a resource for other members in the community who either want to find examples, starting points, or other people to work with. Let me know what you think about this. 

Possioble repo names:
`pupil-community`
`pupil-community-contributions`

**Comment by willpatera** [comment]
@nathakits please make changes as reqested. 

@mkassner let me know what you think about this repo that I suggested above :point_up:

**Comment by mkassner** [comment]
pupil_community is good.

**Comment by willpatera** [comment]
@nathakits I created a new repo at https://github.com/pupil-labs/pupil-community - please update based on this information.

**Pull request merged**